app/components/license-detail/index.ts

```diff
@@ -1,6 +1,9 @@
 import Component from '@glimmer/component';
 import dayjs from 'dayjs';
-import LicenseModel from 'irene/models/license';
+import { inject as service } from '@ember/service';
```

Comment:
You can just import the service directly. 

`import { service } from '@ember/service'`